packages/sdk/src/types/step.types.ts

```diff
@@ -12,7 +12,6 @@ import { MaybePromise } from './util.types';
 export type ActionStepOptions = {
   skip?: Skip;
   inputSchema?: Schema;
-  outputSchema?: Schema;
```

Comment:
not sure why we had it next, [REF](https://github.com/novuhq/echo/pull/1#discussion_r1590334544)